src/components/templates/layout.tsx

```diff
@@ -8,8 +8,8 @@ const Layout: React.FC = ({ children }) => {
       <Sidebar />
       <div className="flex flex-col flex-1">
         <Topbar />
-        <div className="large:px-xlarge py-xlarge bg-grey-5 min-h-content overflow-y-scroll">
-          <main className="xsmall:mx-base small:mx-xlarge medium:mx-4xlarge large:mx-auto large:max-w-7xl large:w-full h-full">
+        <div className="large:px-xlarge py-xlarge bg-grey-5 min-h-content overflow-y-auto">
+          <main className="xsmall:mx-base small:mx-xlarge medium:mx-4xlarge large:mx-auto large:max-w-7xl large:w-full">
```

Comment:
I think this should still have h-full to allow full height children with negative space in components